lua/tests/automated/pickers/entry_display_spec.lua

```diff
@@ -0,0 +1,32 @@
+local entry_display = require('telescope.pickers.entry_display')
+
```

Comment:
That file is in the wrong dir. Everything inside pickers are integration tests. Put it in `lua/tests/automated/`
But thank you very much for writing tests :)

Reply:
Fixed on 45be77c